### PR TITLE
(DOCSP-18744): Make docs reference TLS 1.3

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info
added info about TLS 1.3 and network security to Realm Sync docs. 

### Jira

- https://jira.mongodb.org/browse/DOCSP-18744

### Staged Changes (Requires MongoDB Corp SSO)

- [Security](https://docs-mongodborg-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-18744/security)
- [Sync Protocol](https://docs-mongodborg-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-18744/sync/protocol/#network-security)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sync/protocol.txt [hunk 1]
```diff
@@ -102,6 +102,15 @@ integer strictly less than 2^63.
    free to generate identical identifiers for two client files if they are
    associated with different server files.
 
+.. _network-security: 
+
+Network Security
+~~~~~~~~~~~~~~~~
+
+The SDK syncs with the application server over a WebSocket connection secured by
```
Comment (mongomoe) [inline source/sync/protocol.txt:110]:
```suggestion
The SDK synchronizes with the application server over a WebSocket connection secured by
```

Necessary fix ^, we only use the word "sync" or "syncs" when referring to Realm Sync. This is so users don't get confused about the terminology.

**Review by mongomoe — COMMENTED** [review]
Good stuff, 1 needed fix before LGTM

**Comment by mongodben** [comment]
@mongomoe implemented your suggestion. pls approve

**Review by mongomoe — APPROVED** [review]
Lgtm

**Pull request merged**